eo-parser/src/main/java/org/eolang/parser/xmir/Xmir.java

```diff
@@ -110,4 +110,60 @@ public String toEO() {
                 .get(0);
         }
     }
+
+    /**
+     * Simplified xmir that allows printing of small parts EO.
```

Comment:
@volodya-lombrozo it's not clear, why `Xmir` printing is not enough. Why do we need another printer?